pyqmc/ewald2d.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+import pyqmc
+import pyqmc.energy
+import pyqmc.gpu as gpu
+
```

Comment:
comment at the beginning here that gives the reference that you used, and a description of what this is supposed to do.